project_issue_id_display/README.rst

```diff
@@ -0,0 +1,47 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
```

Comment:
@elicoidal because those 8 module is splited from elico_project module.so ,I didn't create other PR .